libs/SalesforceSDK/res/values/sf__styles.xml

```diff
@@ -2,7 +2,9 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <style name="SalesforceSDK" parent="Theme.AppCompat.DayNight">
+    <style name="SalesforceSDK" parent="Theme.AppCompat.Light.DarkActionBar">
```

Comment:
Now that I really looked over the 10.2 theme for the app bar, it qualifies as "dark" since it always uses white text on blue, regardless of light or dark mode.